### PR TITLE
Increase various test timeouts

### LINKED ISSUE #412: Increase test.PollTimeout

### PR DESCRIPTION
**Pull request by embano1**
Fixes #412

## Proposed Changes

- :broom: Reduce test flakes with increased timeouts/poll intervals
-
-

### Pre-review Checklist

- [x] **At least 80% unit test coverage**
- [x] **E2E tests** for any new behavior
- [ ] **Docs** for any user-facing impact

**Release Note**

```release-note

```

### CODE REVIEW

File: test/clients.go [hunk 1]
```diff
@@ -29,9 +29,9 @@ const (
 	Namespace = "default"
 
 	// PollInterval is how frequently e2e tests will poll for updates.
-	PollInterval = 1 * time.Second
+	PollInterval = 5 * time.Second
```
Comment (embano1) [inline test/clients.go:32]: Be less aggressive, reduce log noise.

File: .github/workflows/kind-e2e.yaml [hunk 2]
```diff
@@ -87,5 +82,11 @@ jobs:
...
         # Run the tests tagged as e2e on the KinD cluster.
-        go test -v -race -timeout=3m -tags=e2e github.com/${{ github.repository }}/test/e2e/...
+        go test -v -race -timeout=10m -tags=e2e github.com/${{ github.repository }}/test/e2e/...
 
+    - name: Debug
+      if: ${{ always() }}
+      run: |
+        kubectl get pods --all-namespaces
+        kubectl -n vmware-sources describe pods
+        kubectl -n vmware-sources get events
```
Comment (gabo1208) [inline .github/workflows/kind-e2e.yaml:92]: missing new line here

File: .github/workflows/kind-e2e.yaml [hunk 3]
```diff
@@ -66,12 +66,7 @@ jobs:
         # Build Knative plugin
         go build -o kn-vsphere ./plugins/vsphere/cmd/vsphere
 
-        kubectl -n vmware-sources wait --timeout=10s --for=condition=Available deploy/vsphere-source-webhook
```
Comment (gabo1208) [inline .github/workflows/kind-e2e.yaml:69]: Maybe have this commented or in the docs for new users to know where to look?

Comment (embano1) [reply .github/workflows/kind-e2e.yaml:69]: I added a comment why this was bumped to `1m`.

**Review by gabo1208 — APPROVED** [review]

**Pull request merged**